api/src/main/resources/jakarta/xml/ws/wsdl_customizationschema_3_0.xsd

```diff
@@ -0,0 +1,415 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Distribution License v. 1.0, which is available at
+    http://www.eclipse.org/org/documents/edl-v10.php.
+
+    SPDX-License-Identifier: BSD-3-Clause
+
+-->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:jaxws="http://xmlns.jakarta.ee/xml/ns/jaxws"
+           elementFormDefault="qualified"
+           targetNamespace="http://xmlns.jakarta.ee/xml/ns/jaxws"
+           attributeFormDefault="unqualified">
+  <xs:annotation>
+    <xs:documentation>
+
+      Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+
+      This program and the accompanying materials are made available under the
+      terms of the Eclipse Distribution License v. 1.0, which is available at
+      http://www.eclipse.org/org/documents/edl-v10.php.
+
+      SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Don't need this twice.

Reply:
This follows the pattern from other EE schemas . The only difference is that those sources are using xinclude (ie [here](https://github.com/eclipse-ee4j/jakartaee-schemas/blob/master/jakartaee9/src/jakartaee_9.xsds#L30)) which I cannot (don't want to) use here

Reply:
...noticed this is not in jaxb bindings schema. So the question here is: should the pattern from other schemas be followed (and this kept/included) or removed (assuming the same in form of xinclude reference is going to be removed from other schemas)?

Reply:
I'm not sure where you're seeing two copyright notices in one file.

I believe we did have an issue at one point where the schemas were processed
as part of a "build" procedure and we wanted to have a copyright on the "source"
file but we also wanted to have a copyright on the output file, so we ended up
with two copyrights in the file, the source file copyright being removed as part
of the build process.  But I think we fixed that by using xinclude to include the
copyright we wanted on the output file, so the source file was back to having
one copyright.

And sometimes people didn't do this right and ended up with two copyrights in
the output file.  Or they used the same source file structure, but with a file that
never went through the "build" process, and so it always had two copyrights for
no reason.

So, I'm not sure why you would have two copyrights in the *.xsd file.  One that's
part of an xsd:annotation/xsd:documentation block is enough.  (I don't remember
why we didn't just do it as an XML comment.)

Reply:
Anyway, just leave one copyright block and this is good to go.

Reply:
Now I see the point. This is `src` and `build-out` at the same time while those others are sources running through `magic` to produce `build-out` thus they need to preserve copyright in some other way.